source/fundamentals/crud/read-operations/retrieve.txt

```diff
@@ -0,0 +1,242 @@
+are useful when your ``Find()`` command returns a large amount of documents.
+
+To find multiple documents in a collection and put them in a ``Cursor``, pass a 
+query filter to the ``Find()`` method that specifies the criteria of the documents 
+you want to find, then append the ``ToCursor`` or ``ToCursorAsync`` method as follows:
+
+.. tabs::
+
+   .. tab:: Asynchronous
+      :tabid: find-cursor-async
+
+      .. code-block:: csharp
+         :copyable: true
+
+         var restaurants = await _restaurantsCollection.Find(filter).ToCursorAsync();
+
+   .. tab:: Synchronous
+      :tabid: find-cursor-sync
+
+      .. code-block:: csharp
+         :copyable: true
+
+         var restaurants = _restaurantsCollection.Find(filter).ToCursor();
+
+Find Documents Using A List
+```````````````````````````
+
+If you are returning a small number of documents, or need your results returned
+as a ``List`` object, use the ``ToList`` or ``ToListAsync`` methods.
+
+To find multiple documents in a collection using a list, pass a query filter 
+to the ``Find()`` method that specifies the criteria of the documents you want 
+to find, then append the ``ToList`` or ``ToListAsync`` method as follows:
+
+.. tabs::
+
+   .. tab:: Asynchronous
+      :tabid: find-list-async
+
+      .. code-block:: csharp
+
+         var restaurants = await _restaurantsCollection.Find(filter).ToList();
+
+   .. tab:: Synchronous
+      :tabid: find-list-sync
+
+      .. code-block:: csharp
+
+         var restaurants = _restaurantsCollection.Find(filter).ToListAsync();
+
+To see a full example of using ``Find()`` to find multiple documents, see
+:ref:`csharp-retrieve-additional-information`.
+
+Find All Documents
+~~~~~~~~~~~~~~~~~~
+
+To find all documents in a collection, pass an empty
+``BsonDocument`` as a query filter. Then append the ``ToList`` or ``ToListAsync``
+method to return the documents as a ``List`` object, or the ``ToCursor`` or ``ToCursorAsync``
+method to return the documents as a ``Cursor``.
+
+To see a fully runnable example of using ``Find()`` to find all documents, see 
+:ref:`csharp-retrieve-additional-information`.
+
+Modify Find Behavior
+~~~~~~~~~~~~~~~~~~~~
+
+You can modify the behavior of ``Find()`` by passing
+in ``FindOptions``. If you don't specify any options, the driver uses the 
+default values for each option.
+
+You can configure the commonly used options with the following methods:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Method
+     - Description
+
+   * - ``BatchSize`` 
+     - | Gets or sets the number of documents to return per batch.
+
+   * - ``Collation`` 
+     - | Sets the collation options. 
+
+   * - ``Comment`` 
+     - | Sets the comment to the query.
+
+   * - ``Hint`` 
+     - | Sets the hint for which index to use.
+       | Default: ``0``
+
+   * - ``MaxTime`` 
+     - | Sets the maximum execution time on the server for this operation.
```

Comment:
Question/Possible Issue: Is there a reason only 1 option is given a default value?

Reply:
This is a good example of the copy/paste errors that we've been talking about lately. I meant to get rid of all the default values. Good Catch! I also removed the reference in the sentence above that talks about these default values.